Source/Core/InputCommon/GCAdapter.cpp

```diff
@@ -307,6 +303,10 @@ static bool CheckDeviceAccess(libusb_device* device)
       ERROR_LOG(SERIALINTERFACE, "libusb_detach_kernel_driver failed with error: %d", ret);
   }
 
+  ret = libusb_control_transfer(s_handle, 0x21, 11, 0x0001, 0, nullptr, 0, 10000);
```

Comment:
What are these magic numbers?  Might be worthwhile to add some constants or some comments

Reply:
Hmm, actually looking closer..this file, has a lot of magic numbers.